### PR TITLE
Closes #206 Add agency standards vignette

### LINKED ISSUE #206: Summary of Differences for Regulatory Agencies
Create a vignette on xportr that lists out differences between the agencies for xpt files.

For example see - https://github.com/atorus-research/xportr/issues/201#issuecomment-1853545157

### PR DESCRIPTION
**Pull request by vedhav**
Closes #206 

### Changes Description

Adds a vignette showing [the standards followed in different agencies](https://github.com/atorus-research/xportr/issues/206#issuecomment-1859763589)

### Task List

- [ ] The spirit of xportr is met in your Pull Request
- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Summary of changes filled out in the above Changes Description. Can be removed or left blank if changes are minor/self-explanatory.
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Use `styler` package and functions to style files accordingly.
- [ ] New functions or arguments follow established convention found in the [Wiki](https://github.com/atorus-research/xportr/wiki/Style-Guide-for-Roxygen-Headers).
- [ ] Updated relevant unit tests or have written new unit tests. See our [Wiki](https://github.com/atorus-research/xportr/wiki/Style-Guide-for-Unit-Tests) for conventions used in this package.
- [ ] Creation/updated relevant roxygen headers and examples. See our [Wiki](https://github.com/atorus-research/xportr/wiki/Style-Guide-for-Roxygen-Headers) for conventions used in this package.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Run `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new/updated functions occur on the "Reference" page.
- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] The `NEWS.md` entry should go under the `# xportr development version` section. Don't worry about updating the version because it will be auto-updated using the `vbump.yaml` CI.
- [ ] Address any updates needed for vignettes and/or templates.
- [ ] Link the issue Development Panel so that it closes after successful merging.
- [ ] The developer is responsible for fixing merge conflicts not the Reviewer.
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by bms63 — COMMENTED** [review]
This is going to be awesome!!  Just pulling in a few random resources if you haven't stumbled upon them already 

https://www.pharmasug.org/proceedings/2021/SS/PharmaSUG-2021-SS-137.pdf
https://www.pharmasug.org/proceedings/2021/EP/PharmaSUG-2021-EP-070.pdf
https://www.lexjansen.com/phuse-us/2019/sa/SA04.pdf

I will track some more down - it also might be a good call out to the slack channel for resources.

File: vignettes/agency_standards.Rmd [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+vignette: >
+  %\VignetteIndexEntry{Standards in Different Regulatory Agencies}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Motivation
+
+The `xportr` package is designed to help clinical programmers create `CDISC` compliant `xpt` files.
+It provides the functionality to associate metadata information to a local R data frame, perform data set level validation checks, and convert into a transport v5 file (`xpt`)
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:16]:
```suggestion
It provides the functionality to associate metadata information to a local R data frame, perform data set level validation checks, and convert into a transport v5 file (`xpt`).
```

File: vignettes/agency_standards.Rmd [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+However, technical requirements related to the `xpt` files can change across different regulatory agencies.
+This vignette aims to provide a clear and concise summary of the differences between the agencies for the `xpt` files.
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:18]:
```suggestion
This vignette aims to start to provide a clear and concise summary of the differences between the agencies for the `xpt` files.  Further updates will come with later package releases.
```

File: vignettes/agency_standards.Rmd [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+
+The following section will delve into various technical specifications as per [FDA](https://www.fda.gov/media/153632/download), [NMPA](https://www.nmpa.gov.cn/directory/web/nmpa/images/obbSqc7vwdm0ssrU0enKb7dtd29u9a4tbzUrdTyo6jK1NDQo6mhty5wZGY=.pdf), and [PMDA](https://www.pmda.go.jp/files/000247157.pdf) guidelines.
+
+
+### File name character
+
+#### XPT <img src="../man/figures/xpt.png" style="height: 34px;"/>
+
+The first character must be an English letter (A, B, C, . . ., Z) or underscore (_). Subsequent characters can be letters, numeric digits (0, 1, . . ., 9), or underscores
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:28]:
```suggestion
The first character must be an English letter (A, B, C, . . ., Z) or underscore (_). Subsequent characters can be letters, numeric digits (0, 1, . . ., 9), or underscores.
```

File: vignettes/agency_standards.Rmd [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+#### FDA <img src="../man/figures/fda.jpg" style="height: 20px;"/>
+
+Dataset in the transport file should be named the same as the transport file.
+Variable names, as well as variable and dataset labels should include American Standard Code for Information Interchange (ASCII) text codes only.
+Dataset names should contain only lowercase letters, numbers, and must start with a letter.
+
+#### NMPA <img src="../man/figures/nmpa.png" style="height: 27px;"/>
+
+The file name and the dataset name must be the same for the SDTM and ADaM datasets
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:41]:
```suggestion
The file name and the dataset name must be the same for the SDTM and ADaM datasets.
```

File: vignettes/agency_standards.Rmd [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+The Japanese dataset and alphanumeric dataset must be identical in structure, except for the data lengths of the Japanese items and the corresponding alphanumeric character sequence
+
+#### PMDA <img src="../man/figures/pmda.png" style="height: 20px;"/>
+
+?
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:46]:
```suggestion
Information has not yet been collected.
```

File: vignettes/agency_standards.Rmd [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+
+8 characters
+
+#### NMPA <img src="../man/figures/nmpa.png" style="height: 27px;"/>
+
+\-
+
+#### PMDA <img src="../man/figures/pmda.png" style="height: 20px;"/>
+
+?
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:66]:
```suggestion
Information has not yet been collected.
```

File: vignettes/agency_standards.Rmd [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+The `xportr` package is designed to help clinical programmers create `CDISC` compliant `xpt` files.
+It provides the functionality to associate metadata information to a local R data frame, perform data set level validation checks, and convert into a transport v5 file (`xpt`)
+However, technical requirements related to the `xpt` files can change across different regulatory agencies.
+This vignette aims to provide a clear and concise summary of the differences between the agencies for the `xpt` files.
+
+
+The following section will delve into various technical specifications as per [FDA](https://www.fda.gov/media/153632/download), [NMPA](https://www.nmpa.gov.cn/directory/web/nmpa/images/obbSqc7vwdm0ssrU0enKb7dtd29u9a4tbzUrdTyo6jK1NDQo6mhty5wZGY=.pdf), and [PMDA](https://www.pmda.go.jp/files/000247157.pdf) guidelines.
+
+
+### File name character
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:24]:
```suggestion
### File name - character
```

File: vignettes/agency_standards.Rmd [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+The file name and the dataset name must be the same for the SDTM and ADaM datasets
+The Japanese dataset and alphanumeric dataset must be identical in structure, except for the data lengths of the Japanese items and the corresponding alphanumeric character sequence
+
+#### PMDA <img src="../man/figures/pmda.png" style="height: 20px;"/>
+
+?
+
+***
+
+### File name length
```
Comment (bms63) [inline vignettes/agency_standards.Rmd:50]:
```suggestion
### File name - length
```

**Review by bms63 — COMMENTED** [review]
Can we replace ? with "Information has not yet been collected."

**Comment by vedhav** [comment]
> Can we replace ? with "Information has not yet been collected."

I just replaced all the nine instances of them in the commit 1b22b3338e5cadaf9b7d6174c7d61f2a47d7c656

**Review by bms63 — APPROVED** [review]
lol i was about to add a comment on the News.md and you just did it!!

This looks awesome!!  Very excited for this to grow.

**Comment by bms63** [comment]
Just a few cosmetics things to fix up and then please merge in.

**Comment by vedhav** [comment]
> Just a few cosmetics things to fix up and then please merge in.

I applied all your changes, Is there anything else you'd like to change or is it ready to merge?

**Comment by bms63** [comment]
> > Just a few cosmetics things to fix up and then please merge in.
> 
> I applied all your changes, Is there anything else you'd like to change or is it ready to merge?

LGTM!!

**Pull request merged**